Tasks/RunDistributedTests/task.json

```diff
@@ -93,6 +98,15 @@
             "required": false,
             "helpMarkDown": "Override parameters defined in the TestRunParameters section of runsettings file. For example: Platform=$(platform);Port=8080",
             "groupName": "testSettings"
+        },
+        {
+            "name": "autEnvironmentName",
+            "type": "string",
+            "label": "Application Under Test Machine Group",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "The name of the Application Under Test Machine Group on which server processes such as W3WP.exe is running.",
```

Comment:
minor : group application under test together .. may be using hyphen ...
so as to avoid confusion in the long word ...  Application Under Test Machine group